pkg/controllers/subnetset/subnetset_controller.go

```diff
@@ -70,8 +73,43 @@ func (r *SubnetSetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		log.Error(err, "Unable to fetch SubnetSet CR", "SubnetSet", req.NamespacedName)
 		return ResultRequeue, err
 	}
+
+	hasBindings := r.subnetSetHasBindings(string(subnetsetCR.UID))
```

Comment:
I prefer to call it as "SubnetBinding" rather than "BindingMap" like I what named the controller and service package. Rename the two functions as `getNSXSubnetBindingsBySubnetSet` and `getSubnetBindingCRsBySubnetSet`